sdk/spring/pom.xml

```diff
@@ -87,4 +87,141 @@
     <module>azure-spring-cloud-stream-binder-test</module>
```

Comment:
All these modules have to be moved to the default profiles section. There should be no modules outside of the profiles section.